### PR TITLE
Fix compat issues with old Test Results publish task with new agent

### PR DESCRIPTION
**Pull request by nigurr**
Issue: There's compat issue between old Test Results publish task with new agent. 
Initial call is to break the compat and move to right solution, however to avoid customer issues, keeping the compat

Fix: accept the old way of vsts-task-lib command

### CODE REVIEW

**Comment by nigurr** [comment]
Manual testing done

File: src/agent/commands/results.publish.ts [hunk 1]
```diff
@@ -37,20 +37,10 @@ export class ResultsPublishCommand implements cm.IAsyncCommand {
         var publishRunAttachments: boolean = (this.command.properties['publishRunAttachments'] === "true");
         var resultFilesPath = this.command.properties['resultFiles'];
         var mergeResults: boolean = (this.command.properties['mergeResults'] === 'true');
+        var fileNumber: string = this.command.properties['fileNumber'];
```
Comment (acesiddhu) [inline src/agent/commands/results.publish.ts]: do we know why this was missed? and we can do to catch this earlier?

Comment (nigurr) [inline src/agent/commands/results.publish.ts]: It's not missed. It's no longer require by new way of handling this task. it's only required for old task compat

**Comment by acesiddhu** [comment]
looks fine

**Comment by AshwiniChalla** [comment]
approved

**Pull request merged**